src/app.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8" />
-		<link rel="icon" href="%sveltekit.assets%/images/favicon.png" />
+		<link rel="icon" href="https://deepseatactics.com/images/favicon.png" />
```

Comment:
This should be %sveltekit.assets% (sveltekit automatically inlines this, and the directory is hashed for caching purposes).

Reply:
done